Dockerfile

```diff
@@ -22,9 +22,6 @@ RUN git clone --depth=1 https://github.com/k8snetworkplumbingwg/multus-cni && \
 
 # Create the multus image
 FROM scratch
-COPY --from=builder  /go/multus-cni/bin /usr/src/multus-cni/bin
-COPY --from=builder  /go/multus-cni/LICENSE /usr/src/multus-cni/LICENSE
```

Comment:
Do we need to also copy the `LICENSE` file for distribution in the built container image?